src/project/qml/MuseScore/Project/internal/Export/AudioSettingsPage.qml

```diff
@@ -23,67 +23,73 @@ import QtQuick 2.15
 import QtQuick.Layouts 1.15
 
 import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
 import MuseScore.Project 1.0
 
-ColumnLayout {
+ExportSettingsPage {
     id: root
-    spacing: 12
-
-    property ExportDialogModel model
-    property int firstColumnWidth
 
     property bool showBitRateControl: false
 
     CheckBox {
+        width: parent.width
         text: qsTrc("project", "Normalize")
+
+        navigation.name: "NormalizeAudioCheckbox"
+        navigation.panel: root.navigationPanel
+        navigation.row: root.navigationOrder + 1
+
         checked: root.model.normalizeAudio
         onClicked: {
             root.model.normalizeAudio = !checked
         }
     }
 
     ExportOptionItem {
-        Layout.fillWidth: false
         text: qsTrc("project", "Sample rate:")
-        firstColumnWidth: root.firstColumnWidth
 
         Dropdown {
-            id: srates
             Layout.preferredWidth: 126
 
+            navigation.name: "SampleRatesDropdown"
```

Comment:
You also need to add the option title(Format) for accessibility, because the reader will only say the value
and it will not be clear to the user

navigation.accessibility.name: option.text + " " + currentText

Reply:
It's difficult that I can't really test it, because our accessibility system does not work at all on macOS (#9153).
Anyway, I did now what seemed right to me, so I hope that will work.